.travis.yml

```diff
@@ -1,14 +1,34 @@
 dist: trusty
-sudo: false
 language: node_js
 node_js:
   - "6.10.0"
+
+sudo: required
+
+services:
+  - docker
+
+env:
+ - DOCKER_COMPOSE_VERSION=1.4.2
+
 cache:
   directories:
     - ui/node_modules
 
-script: cd ui && npm install && npm test
+script: 
+  - cd ui && npm install && npm test
+
 before_install:
   - export CHROME_BIN=chromium-browser
   - export DISPLAY=:99.0
-  - sh -e /etc/init.d/xvfb start
\ No newline at end of file
+  - sh -e /etc/init.d/xvfb start
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
```

Comment:
I don't think we need to install docker composer here. because we do not do anything with it, at least for now.